CHANGELOG.asciidoc

```diff
@@ -28,9 +28,28 @@ Notes:
         ==== x.x.x - YYYY/MM/DD
 ////
 
+
 [[release-notes-3.x]]
 === Node.js Agent version 3.x
 
+==== Unreleased
+
+[float]
+===== Breaking changes
+
+[float]
+===== Features
+
+[float]
+===== Bug fixes
+
+[float]
+===== Chores
+
+* Move configuration to its own module and add normalizer tests.
+  ({pull}3358[#3358])
```

Comment:
FWIW, I think if a change is completely internal and shouldn't impact users at all, then it would be fine to not mention it in the changelog.

Reply:
I agree, let's remove it.